core/web/eth_keys_controller.go

```diff
@@ -355,10 +355,11 @@ func (ekc *ETHKeysController) setLinkBalance(ctx context.Context, state ethkey.S
 	} else {
 		ethClient := chain.Client()
 		addr := common.HexToAddress(chain.Config().LinkContractAddress())
-		bal, err = ethClient.GetLINKBalance(ctx, addr, state.Address.Address())
+		linkBalance, err := ethClient.LINKBalance(ctx, state.Address.Address(), addr)
 		if err != nil {
 			ekc.lggr.Errorw("Failed to get LINK balance", "chainID", chainID, "address", state.Address, "error", err)
 		}
+		bal = (*assets.Link)(linkBalance)
```

Comment:
can we avoid casting here?

Reply:
yes, and same as @dimriou 's comment.You can just directly use linkBalance in the return line 364. No need to cast it, as it is already of that type.

Reply:
Yep, this piece of code went through the series of changes when `TokenBalance` call was used and required explicit casting. Fixed.